CharacterCreation/02_Classes.txt

```diff
@@ -366,27 +367,83 @@ Skill Proficiencies:
 * Weapons - General
 
 == Class Feats ==
-	----------------------
-	Planes, Trains, Automobiles -
-	This is sorta familiar to something you once saw, or rode on. Maybe? 
-	Unfamiliar vehicles can be operated [albeit pretty poorly] after you pass 
-	a DC 50 Luck check, or take up to 3 actions examining.
-	|
-	|
-	V
-	You have flown, operated and driven just about everything there is. Except 
-	that. Make a DC 50 Piloting OR Luck check, or take up to 2 actions examining.
-	----------------------
-	Extensive training <Vehicle> -
+
+Getaway Driver:
+
+    You have an intuitive understanding of the flow of traffic and the lines of
+sight between your vehicle and those you'd rather avoid. Maybe you had a 
+criminal career in the past, or maybe you just see the road differently then
+most. 
+
+    Any enemy that targets the vehicle that you are piloting, the passengers in
+the vehicle that you are piloting or you whilst you are piloting takes a -2 to
+accuracy rolls to hit as long as the vehicle has moved for at least 2 Actions
+on your last turn. Furthermore you get a +12 to Stealth (Dex) rolls made to lose 
+a tail or Piloting(Dex) rolls to make maneuvers in retreating.
+
+Wheelman:
+    
+    You have mastered the zen of the flow of vehicle combat. You don't just know
+when to break line of sight to avoid harm, but when to hold her steady so your
+teammates can light 'em up. You also have a concerning talent for ending enemy
+lives with the blunt force of your bumper.
+
+    Teammates firing from a vehicle that you are piloting get a +1 to accuracy
+rolls and negate any existing penalties from chaotic vehicle movement. If you
+attempt to make an attack by ramming, crashing, scraping or splattering, you 
+gain a +2 to hit with the attack roll and any damage to your vehicle from your
+attacks is reduced to three quarters. 
+
+    This Class Feat requires a Character to be level 10 and to have taken 
+Getaway Driver before they can take it.
```

Comment:
nitpick: "before they can take it" isn't strictly necessary to the meaning of the sentence.